src/plugins/plugin.legend.js

```diff
@@ -369,10 +369,10 @@ var Legend = Element.extend({
 				if (opts.labels && opts.labels.usePointStyle) {
 					// Recalculate x and y for drawPoint() because its expecting
 					// x and y to be center of figure (instead of top left)
-					var radius = fontSize * Math.SQRT2 / 2;
-					var offSet = radius / Math.SQRT2;
-					var centerX = x + offSet;
-					var centerY = y + offSet;
+					var pointWidth = Math.min(boxWidth, fontSize);
```

Comment:
Because `boxWidth > fontSize` is checked in `getBoxWidth`, you can remove this line and use `boxWidth` instead of `pointWidth` in calculations below.

Reply:
I missed that, tested and committed changes.